src/main/kotlin/graphics/scenery/backends/vulkan/VulkanRenderer.kt

```diff
@@ -1287,8 +1323,178 @@ open class VulkanRenderer(hub: Hub,
         }
     }
 
+    var vdiStreaming: Boolean = false
```

Comment:
There's no good reason for this to be in the rendering code, as that also violates separation of concerns. I'd argue to move this to the vdi package.

Reply:
With the postRenderLambdas, the necessary infrastructure is also available to factor this out of the core rendering code.